frontend/packages/knative-plugin/yamls/knative-serving/smmr.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
```

Comment:
Should we create a special namespace and add that namespace to this member role? We typically discourage people from using `default` to deploy apps.

Reply:
makes sense @sbose78 any suggestions or shall i go with `mynamespace` as in docs https://docs.openshift.com/container-platform/4.1/serverless/installing-openshift-serverless.html#installing-service-mesh-member-roll_installing-openshift-serverless